scripts/run-tests.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Get the root directory (parent of scripts directory)
+ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+
+# Change to .docker-magento directory
+cd "$ROOT_DIR/.docker-magento"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)